Source/App.js

```diff
@@ -1 +1,417 @@
-var cesiumWidget = new Cesium.Viewer('cesiumContainer');
+    //////////////////////////////////////////////////////////////////////////
+    // Viewer configuration
+    //////////////////////////////////////////////////////////////////////////
+
+    var viewer = new Cesium.Viewer('cesiumContainer', {
+        navigationHelpButton: false,
+        scene3DOnly: true,
+        selectionIndicator: false,
+        baseLayerPicker: false
+    });
+
+    // Set the initial camera view
+    var initialPosition = Cesium.Cartesian3.fromDegrees(-74.01881302800248, 40.69114333714821, 753);
+    var initialOrientation = new Cesium.HeadingPitchRoll.fromDegrees(21.27879878293835, -21.34390550872461, 0.0716951918898415);
+    viewer.scene.camera.setView({
+        destination: initialPosition,
+        orientation: {
+            heading: initialOrientation.heading,
+            pitch: initialOrientation.pitch,
+            roll: initialOrientation.roll
+        }
+    });
+
+    //Override the default home button
+    viewer.homeButton.viewModel.command.beforeExecute.addEventListener(function (arg) {
+        arg.cancel = true;
+        viewer.scene.camera.flyTo({
+            destination: initialPosition,
+            orientation: {
+                heading: initialOrientation.heading,
+                pitch: initialOrientation.pitch,
+                roll: initialOrientation.roll
+            },
+            endTransform: Cesium.Matrix4.IDENTITY,
+            duration: 1.0
+        });
+    });
+
+    //Set up clock and timeline.
+    viewer.clock.shouldAnimate = true;
+    viewer.clock.clockStep = Cesium.ClockStep.TICK_DEPENDENT;
+    viewer.clock.clockRange = Cesium.ClockRange.LOOP_STOP; // loop at the end
+    viewer.clock.multiplier = 60; // sets a speedup
+    viewer.clock.startTime = Cesium.JulianDate.fromIso8601("2016-08-01T04:00:00Z");
+    viewer.clock.stopTime = Cesium.JulianDate.fromIso8601("2016-08-02T04:00:00Z");
+    viewer.clock.currentTime = Cesium.JulianDate.fromIso8601("2016-08-01T16:00:00Z");
+    viewer.timeline.zoomTo(viewer.clock.startTime, viewer.clock.stopTime); // set visible range
+
+    //Enable lighting based on sun/moon positions
+    viewer.scene.globe.enableLighting = true;
+
+    //////////////////////////////////////////////////////////////////////////
+    // Loading Imagery
+    //////////////////////////////////////////////////////////////////////////
+
+    Cesium.BingMapsApi.defaultKey = 'AsarFiDvISunWhi137V7l5Bu80baB73npU98oTyjqKOb7NbrkiuBPZfDxgXTrGtQ'; // For use on cesiumjs.org only
```

Comment:
Right, I sort of put this in as a placeholder because I wasn't sure -- what's the correct thing to do re Bing key for this application? Should we request a new one just for this repo? @hpinkos